pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -586,6 +588,7 @@ type Disk struct {
 	FilesystemOverhead *cdiv1.Percent `xml:"filesystemOverhead,omitempty"`
 	Capacity           *int64         `xml:"capacity,omitempty"`
 	ExpandDisksEnabled bool           `xml:"expandDisksEnabled,omitempty"`
+	Shareable          *Shareable     `xml:"shareable,omitempty"`
```

Comment:
What is the purpose of this? I don't see it referenced anywhere.

Reply:
This is needed for define the field in the XML. It is used for example in https://github.com/alicefr/kubevirt/blob/shareable/pkg/virt-launcher/virtwrap/converter/converter.go#L191